object-alloc/README.md

```diff
@@ -0,0 +1,10 @@
+<!-- Copyright 2017 the authors. See the 'Copyright and license' section of the README.md file at the top-level directory of this repository.
+
+Licensed under the Apache License, Version 2.0 (the LICENSE file). This file may not be copied, modified, or distributed except according to those terms. -->
+
+object-alloc
+============
+
+Object allocators in Rust. This crate defines the `ObjectAlloc` trait and related types. An object allocator is an allocator which allocates and caches objects of a particular type, allowing for a number of performance improvements over general-purpose allocators that need to be able to service allocation requests of any size and alignment.
```

Comment:
I don't think it's obvious that the term "caches" means "object constructors can be elided". It may be worth spelling that out here

Reply:
Good point. I'll do that.